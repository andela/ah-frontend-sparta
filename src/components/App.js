import React from 'react';
import Routes from './Routes';
import '../assets/scss/style.scss';

const App = () => (
  <>
    <Routes />
  </>

);

export default App;
