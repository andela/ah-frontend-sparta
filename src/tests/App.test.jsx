import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../store';
import App from '../components/App/index';

describe('App', () => {
  it('should render without crashing', () => {
    const wrapper = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
