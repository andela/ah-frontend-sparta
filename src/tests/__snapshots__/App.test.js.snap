// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
Array [
  <div
    className="nav"
  >
    <div
      className="nav-logo"
    >
      <img
        alt="site logo"
        src={Object {}}
      />
    </div>
    <div
      className="nav-menu"
    >
      <ul>
        <li>
          <form>
            <input
              placeholder="SEARCH"
              type="text"
            />
          </form>
        </li>
        <li
          className="nav-item dropdown"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            className="nav-link dropdown-toggle"
            data-toggle="dropdown"
            href="#"
            role="button"
          >
            Accounts
          </a>
          <div
            className="dropdown-menu"
          >
            <a
              className="dropdown-item"
              href="/register"
            >
              Register
            </a>
            <a
              className="dropdown-item"
              href="/login"
            >
              Login
            </a>
          </div>
        </li>
      </ul>
    </div>
  </div>,
  <div>
    <div
      className="page-banner-image"
    >
      <img
        alt="Landing image"
        className="LandingImage"
        src={Object {}}
      />
    </div>
    <div
      className="small-screen"
    >
      <div>
        <form>
          <input
            placeholder="SEARCH"
            type="text"
          />
        </form>
      </div>
      <div>
        <a
          href="/login"
        >
          Account
        </a>
      </div>
    </div>
    <div
      className="wrapper"
    >
      <div>
        <div>
          No Posts where Found !!!!
        </div>
      </div>
      <div>
        <h1>
          Trending
        </h1>
        <br />
        <hr />
        <br />
        <div>
          No Trending Articles where Found !!!!
        </div>
      </div>
    </div>
    <br />
    <br />
    <br />
  </div>,
]
`;
